packages/component-library/src/CivicSandboxMap/CivicSandboxTooltip.js

```diff
@@ -52,20 +47,10 @@ const MapTooltip = (props) => {
       { tooltipContent }
     </div>
   );
-
-  const tooltipRender = keyValuePairs.length > 0 ? tooltipWrapper : null;
-
-  return (
-    <div>
-      { tooltipRender }
-    </div>
-  );
 };
 
 MapTooltip.propTypes = {
-  tooltipInfo: PropTypes.object,
-  x: PropTypes.number,
-  y: PropTypes.number,
+  tooltipData: PropTypes.object,
```

Comment:
Maybe adding an object shape here could assist in reasoning about what keyValuePairs is doing?